tests/api/Jalasoft.TeamUp.Projects.API.Tests/GetProjectsTest.cs

```diff
@@ -34,5 +35,20 @@ public void GetProject_Returns_OkObjectResult()
             var okObjectResult = Assert.IsType<OkObjectResult>(response);
             Assert.IsType<Project[]>(okObjectResult.Value);
         }
+
+        [Fact]
+        public void GetProjectsBySkill_Returns_OkObjectResult()
```

Comment:
please change the name according to https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices which is the convention used in the project